### DISCUSSION TITLE
How to use zustand useStore.subscribe() api

### DISCUSSION THREAD

**Discussion by laboonly** [udecode/zustand-x#91] [Q&A] [answered]
**Description**

I want to monitor store changes and then perform some operations. I used the useStore.subscribe API, but it reported an error saying that this method could not be found.

**Steps**

export const useStore = () => mapValuesKey('use', rootStore);

const saveStore = useStore.subscribe(console.log('saveStore'));

**Expectation**

I want to know how to use useStore.subscribe api

**Comment by yf-yang** [comment, marked as answer]
`useStore.store.subscribe` is available
